Installation/CHANGES.md

```diff
@@ -161,6 +188,22 @@ Release date: April 2018
 
     [METIS library]: http://glaros.dtc.umn.edu/gkhome/metis/metis/overview
 
+### IO Streams
+
+-   Added new functions to support some parts of the WKT file format:
+    - `CGAL::read_point_WKT()`
+    - `GCAL::read_multi_point_WKT()`
+    - `GCAL::read_linestring_WKT()`
+    - `GCAL::read_multi_linestring_WKT()`
+    - `GCAL::read_polygon_WKT()`
+    - `GCAL::read_multi_polygon_WKT()`
+    - `CGAL::write_point_WKT()`
+    - `CGAL::write_polygon_WKT()`
+    - `CGAL::write_linestring_WKT()`
+    - `CGAL::write_multi_point_WKT()`
+    - `CGAL::write_multi_polygon_WKT()`
+    - `CGAL::write_multi_linestring_WKT()`
+    - `CGAL:read_WKT()`
```

Comment:
That is wrong, because here you modify the part of the CHANGES.md of CGAL-4.12, whereas this PR will be in 4.13.